### PR TITLE
Adjust AWS configuration to use stored files

### PR DESCRIPTION
**Pull request by stephen-lt-sl**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -6,10 +6,11 @@ var verifier = require("google-id-token-verifier");
 var fs = require("fs");
 var mkdirp = require("mkdirp");
 
+var serverIpAddress = process.env.SERVER_IP_ADDRESS || "127.0.0.1";
 var port = process.env.PORT || 8080;
 var oauthClientId = "627385202945-oqedl0onib41h39quc15pufqgqp8j8cu.apps.googleusercontent.com";
```
Comment (susanthomson) [inline server.js]: Maybe this pull request is a good place to make this an environment variable so we can use separate production and dev oauth credentials. Potential for faf though so your call.

Comment (stephen-lt-sl) [reply server.js]: Yeah, probably a good call - everyone's gonna need to set the variable on their own systems after this though

**Comment by susanthomson** [comment]
:+1: I have the production credentials when you're ready for them.

**Comment by stephen-lt-sl** [comment]
dw, all the credentials and config files are already on the AWS instance, so it should be good to go as soon as we merge in

**Comment by susanthomson** [comment]
Actually now that I think about it I can probably just add the production url as authorised origin and redirect urls to the existing credentials.

**Comment by susanthomson** [comment]
Should server/config go back in the .gitignore?

**Review by susanthomson — APPROVED** [review]

**Pull request merged**